server/views/pages/caseAdmin/initialChecks/tasklist.njk

```diff
@@ -43,19 +42,20 @@
                 items: eligiblityChecks
             }) }}
 
+            {% set suitabilityBlocked = criteria.eligibilityStatus != 'ELIGIBLE' and suitabilityStatus === 'NOT_STARTED' %}   
```

Comment:
should this be !== or doesn't it matter?

Reply:
yes I think so, I'll do in a follow up